playground/tailwind-sourcemap/postcss.config.js

```diff
@@ -1,5 +1,3 @@
-console.log(__dirname + '/tailwind.config.js')
-
```

Comment:
This `console.log` was added by https://github.com/vitejs/vite/pull/8869/files#diff-c6ee19322f3186ea82b36cb3d23dcee22d5e6fff71e861b5fb908403f5f5a79a
I guess this is not used.